demo/remote/gcp/terraform/control/.terraform.lock.hcl

```diff
@@ -0,0 +1,89 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
should this be in .gitignore?

Reply:
huh, apparently so: https://www.terraform.io/docs/language/dependency-lock.html
> Terraform has made some changes to the provider dependency selections recorded
> in the .terraform.lock.hcl file. Review those changes and commit them to your
> version control system if they represent changes you intended to make.

TIL.

Reply:
So it should _not_ be in .gitignore (i.e. having it in the repo is the right thing to do)?

Reply:
You are correct.